File: particle-connect/android/src/main/java/com/particleconnect/model/InitData.java
package com.particleconnect.model;

import com.google.gson.annotations.SerializedName;
import com.particle.base.model.DAppMetadata;


public class InitData {

    @SerializedName("chain_name")
    public String chainName;

    @SerializedName("chain_id")
    public long chainId;

    @SerializedName("chain_id_name")
    public String chainIdName;

    @SerializedName("env")
    public String env;

    @SerializedName("metadata")
    public DAppMetadata metadata;

    @SerializedName("rpc_url")
    public RpcUrl rpcUrl;
}

File: particle-aa/android/src/main/java/com/particleaa/model/InitData.kt
package com.particleaa.model

import com.google.gson.annotations.SerializedName

class InitData {
    @SerializedName("chain_name")
    var chainName: String? = null

    @SerializedName("chain_id")
    var chainId:Long = 0


    @SerializedName("env")
    var env: String? = null
}
